components/spinner.scss

```diff
@@ -0,0 +1,4 @@
+$fd-fonts-path: "../scss/fonts/";
```

Comment:
I don't think this component needs fonts to work

Reply:
no, we dont need fonts, thanks for point out.